app/src/main/java/com/door43/translationstudio/ui/translate/TargetTranslationActivity.java

```diff
@@ -618,24 +619,24 @@ public void onClick(View v) {
      */
     @Deprecated
     private void setFocusOnTextSearchEdit() {
-        Handler hand = new Handler(Looper.getMainLooper());
-        hand.post(new Runnable() {
-            @Override
-            public void run() {
-                final EditText edit = (EditText) findViewById(R.id.search_text);
-                edit.setFocusableInTouchMode(true);
-                edit.requestFocus();
-
-                Handler hand = new Handler(Looper.getMainLooper());
-                hand.post(new Runnable() {
-                    @Override
-                    public void run() {
-                        App.showKeyboard(TargetTranslationActivity.this, edit, true);
-                    }
-                });
+        if(mSearchEditText != null) {
+            Handler hand = new Handler(Looper.getMainLooper());
+            hand.post(new Runnable() {
+                @Override
+                public void run() {
+                    mSearchEditText.setFocusableInTouchMode(true);
+                    mSearchEditText.requestFocus();
 
-            }
-        });
+                    Handler hand = new Handler(Looper.getMainLooper());
```

Comment:
Why are we executing another handler? We are already on the main thread. See line 623 above.